### PR TITLE
Added channel info to events

### PR DESCRIPTION
**Pull request by wbenoit26**

### CODE REVIEW

File: projects/online/online/utils/gdb.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def submit(self, event: Event):
         logging.info(f"Submitting trigger to file {event.filename}")
         event_dir = self.write_dir / f"event_{int(event.gpstime)}"
         filename = event_dir / event.filename
-        response = self.createEvent(
+        response = self.create_event(
```
Comment (EthanMarx) [inline projects/online/online/utils/gdb.py:34]: Nice - GraceDB has exposed these pythonic names to the API as well?

File: projects/online/online/utils/gdb.py [hunk 2]
```diff
@@ -63,31 +63,31 @@ def submit_pe(
...
-        self.writeLog(
+        self.write_log(
             graceid,
             "Mollview projection",
             filename=mollview_fname,
             tag_name="sky_loc",
         )
 
-        self.writeLog(graceid, "O3 Replay")
+        self.write_log(graceid, "O3 Replay")
```
Comment (EthanMarx) [inline projects/online/online/utils/gdb.py:79]: We can get rid of this now, right?

**Review by EthanMarx — CHANGES_REQUESTED** [review]

File: projects/online/online/utils/gdb.py (hunk 1 above)
Comment (wbenoit26) [reply projects/online/online/utils/gdb.py:34]: Yeah, both are valid

File: projects/online/online/utils/gdb.py (hunk 2 above)
Comment (wbenoit26) [reply projects/online/online/utils/gdb.py:79]: Good point, I'll get rid of that

**Review by EthanMarx — APPROVED** [review]

**Pull request merged**